themes/wporg-translate-events-2024/blocks/components/event/list.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Wporg\TranslationEvents\Theme_2024;
+use Wporg\TranslationEvents\Translation_Events;
+
+
+register_block_type(
+	'wporg-translate-events-2024/component-event-list',
+	array(
+		// phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.Found
+		'render_callback' => function ( array $attributes ) {
+			$event_ids = $attributes['event_ids'] ?? array();
+			if ( empty( $event_ids ) ) {
+				return;
+			}
+			ob_start();
+			?>
+			<div class="wp-block-wporg-event-list">
+			<ul class="wporg-marker-list__container">
+				<?php
+				foreach ( $event_ids as $event_id ) {
+					$event = Translation_Events::get_event_repository()->get_event( $event_id );
```

Comment:
I think we should add a new `get_events( array $event_ids ): Event[]` method to the repository, so that we don't do one query per event.